### ISSUE TITLE
Variables with labels longer than 255 characters cause the program to hang in infinit while loop.

### ISSUE THREAD

**Issue by davechez** [tiamo/spss#32]
I figured out my issue. The problem is that there is a bug in the code and it guarantees that any variable with a label that has a length longer than 255 will cause an infinite while loop. The code in the Record\Variable class at line 147 is this:

```php
while ($labelLengthBytes > 255) {
    // Strip one char, can be multiple bytes
    $label = mb_substr($this->label, 0, -1);
    $labelLengthBytes = mb_strlen($label, '8bit');
}
```

You can see that $this->label never changes so the loop will go on forever. Here is the fix:

```php
while ($labelLengthBytes > 255) {
    // Strip one char, can be multiple bytes
    $label = mb_substr($label, 0, -1);
    $labelLengthBytes = mb_strlen($label, '8bit');
}
```

The fix is to update line the "$this->label" to "$label" on line 149 so the label length actually decreases in length. If someone from the team can confirm my suspicions are correct I can submit a quick PR.

**Comment by mennodekker** [comment]
I have seen this before and thought it was already fixed but you are right, please submit a PR.

**Comment by buyPcToJJboom** [comment]
I submitted this pull request, but it was closed
https://github.com/tiamo/spss/pull/19

**Comment by mennodekker** [comment]
Thats why it looked so familiar... @tiamo can you please merge the pull request #19 or explain why you closed it without merging?

**Comment by tiamo** [comment]
Sorry guys, I thought it was changed earlier and I closed that PR.. I have merged the pull request #19.

**Comment by tiamo** [comment]
Thank for your contribution!
